lib/declaration.ts

```diff
@@ -422,13 +426,11 @@ class DeclarationTranspiler extends base.TranspilerBase {
     this.visitDecorators(decl.decorators);
     this.visitEachIfPresent(decl.modifiers);
 
-    // Temporarily deactivated to make migration of Angular code base easier.
-    return;
-
     if (this.hasFlag(decl.modifiers, ts.NodeFlags.Protected)) {
       this.reportError(decl, 'protected declarations are unsupported');
       return;
     }
+    if (!decl.name) return;
```

Comment:
I don't remember what was causing it when I put the check in, but I took it out again and the tests all pass, so I'll just leave it out